tests/test_ext_autodoc.py

```diff
@@ -161,7 +161,6 @@ def g(a='\n'):
         pass
     assert formatsig('function', 'f', f, None, None) == '(a, b, c=1, **d)'
     assert formatsig('function', 'f', f, 'a, b, c, d', None) == '(a, b, c, d)'
-    assert formatsig('function', 'f', f, None, 'None') == '(a, b, c=1, **d) -> None'
```

Comment:
There are no way to pass retann value directly without args in actual usecase. So I removed this.